manifests/webhook/package.pp

```diff
@@ -1,19 +1,13 @@
 # Private class, do not include it directly.
 # Installs the webhook packages
 class r10k::webhook::package (
-  $is_pe_server = $r10k::params::is_pe_server,
-  $provider     = $r10k::params::provider,
+  $is_pe_server    = $r10k::params::is_pe_server,
+  $provider        = $r10k::params::provider,
+  $sinatra_version = $r10k::params::webhook_sinatra_version,
 ) inherits r10k::params {
   if !defined(Package['sinatra']) {
     package { 'sinatra':
-      ensure   => installed,
-      provider => $provider,
-      before   => Service['webhook'],
-    }
-  }
-  if !defined(Package['rack']) {
-    package { 'rack':
-      ensure   => installed,
+      ensure   => $sinatra_version,
```

Comment:
As `~> 1.0` may not work for all possible providers, though it does work with the default provider, can you add some verbiage about it this param in either this file's header as comments, or in the README?